src/peft/utils/other.py

```diff
@@ -395,6 +395,12 @@ def _prepare_prompt_learning_config(peft_config, model_config):
             raise ValueError("Please specify `num_attention_heads` in `peft_config`")
         peft_config.num_attention_heads = num_attention_heads
 
+    if "num_key_value_heads" in model_config:
```

Comment:
Could you please add a comment here: `# For grouped-query attention, see #1901`.